gulpfile.js

```diff
@@ -38,7 +40,7 @@ gulp.task('coveralls', function () {
 gulp.task('testCI', ['lint', 'style', 'cover'], function () {
   if (process.env.NODE_ENV !== 'test') {
     gulp.src(process.env.COVERAGE_DIR + '/coverage')
-      .pipe(plugins.clean());
+      .pipe(vinylPaths(del));
```

Comment:
Even simpler would be to just use `fs` and delete the folder entirely, but I figured you were looking to keep with the gulp convention.